Вселенная RimWorld/cryptosleep_revival_briefing_trans.txt

```diff
@@ -0,0 +1,147 @@
+
+    Планетарные прогрессия и регрессия
+
+Общирные пропасти пространства и времени между звёздами делают отдельные миры уязвимыми для регрессии.
+В ваше время (пять столетий после промышленной революции) многие видели технологический прогресс как неумолимое явление жизни. Это не так. Дайте достаточно времени, и почти любая планетарная культура подвергнется стихийному бедствию, эпидемии, войне, либо культурному перевороту, что отбросит её в развитии на тысячелетия, а то и полностью изменит.
+Многие из наших планет погрязли в средневековых распрях и тотчас встали перед Мальтузианским лимитом популяции на тысячи лет. (ком.пер: Malthusian population limits = ограничение на рост народонаселения из-за недостаточности производства, в часности производства пищи) Кто-то развился до раннеиндустиального уровня, а затем оказались застопорены из-за идеологий отрицания технологии (ideological opposition to technology), недостатка ресурсов или постоянной войны.
+Ядерный век - жестокий вызов для каждого мира. Примерно половина культур «вбомбили себя в каменный век» в первые 50 лет после открытия атомной энергии (использовано выражение, неожиданно часто появляющееся в мирах на этой стадии развития). Затем приходят испытания общедоступной биоинженерией, микроскопическими механитами (mechanites), чипами счастья (joywires), сотовыми аккумуляторами (hex-cell energy storage) и ИИ-личностями (AI persona), приведшее к уничтожению тысяч людей каждое.
+Некоторые планеты предпочли не рисковать с подобным. Изучив записи Ордо Историа, всё большее число миров ограничивают себя доядерными технологиями. У некоторых это даже получается в течении нескольких столетий. Но даже эти попытки луддизма в конечном итоге в основном терпят неудачу, когда некое меньшинство получает преимущество, используя запрещённые технологии (прим.пер. гонка технологий).
+
+    Типы миров
+
+По состояниям, в которых пребывает планета, их неформально делят на:
+  * Мёртвые миры: Планеты, в значительной степени не тронутые человеческой рукой. По большей части необитаемые. До первого прибытия людей это любая планета.
+  * Животные миры: Планеты без населения. Либо все умерли, либо планета засеяна при помощи терраформирующих роботов растительной и животной жизнью, но никто не прибыл.
+  * Средневековые миры: Подобны Земле в период от аграрной до промышленной революций. Общественный строй (Social structures) обычно феодальный, либо империальный. Планеты могут находится в таком состоянии тысячелетия. (NB находится в состояни-и можно тысячелетия и тысячелетиями, но сохранять состояни-е можно только тысячелетиями)
+  * Миры пара: Подобны Земле 19 столетия. Зачастую это состояние недолговечно, так как общество развивается до среднего мира, но может и очень сильно затянуться, в зависимости от культурной и правленческой структуры. (прим.пер. моддеры, стимпанк не противоречит лору)
```

Comment:
> Steamworlds: Similar to Earth in the 19th century. Often this state is short-lived, as societies develop into midworlds, but it can be very stretched out depending on culture and government structure.

В итальянском переводе (другого нет) "Mondi industriali". Оригинальная терминология поменялась (крайне навряд ли? что оба переводчика посчитали steam аналогом industial) . Или лучше быть радикально консервативными и плевать на это?
 